backend/base/templates/cancelled_organization.html

```diff
@@ -1,250 +1,167 @@
+                        </td>
+                        <td>{{ organization.phone_number }}</td>
+                        <td>{{ organization.get_status_display }}</td>
+                      </tr>
+                    {% endif %}
+                  {% endfor %}
+                </tbody>
+              </table>
             </div>
           </div>
-        </footer>
-      </div>
+          <a href="{% url 'tenantuser_page' %}" class="btn btn-primary mt-4" data-toggle="tooltip" title="Back" style="border-radius: 50%; height: max-content; width: max-content; margin-left: auto; margin-right: auto; display: block;">
+            <i class="fas fa-arrow-left"></i>
+          </a>
+        </div>
+      </main>
+      <footer class="py-4 bg-light mt-auto">
+        <div class="container-fluid px-4 d-flex align-items-center justify-content-center">
+          <div class="text-muted">Copyright &copy; Your Website 2023</div>
+        </div>
+      </footer>
     </div>
-    <script
-      src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"
-      crossorigin="anonymous"
-    ></script>
-    <!--   -->
-    <script
-      src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.8.0/Chart.min.js"
-      crossorigin="anonymous"
-    ></script>
-    <!--  
-           -->
-    <script
-      src="https://cdn.jsdelivr.net/npm/simple-datatables@7.1.2/dist/umd/simple-datatables.min.js"
-      crossorigin="anonymous"
-    ></script>
-    <!--    -->
+  </div>
 
-    <script>
-      window.addEventListener("DOMContentLoaded", (event) => {
-        // Simple-DataTables
-        // https://github.com/fiduswriter/Simple-DataTables/wiki
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.8.0/Chart.min.js" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/simple-datatables@7.1.2/dist/umd/simple-datatables.min.js" crossorigin="anonymous"></script>
+  <script>
+    document.addEventListener("DOMContentLoaded", () => {
+      // Initialize Simple-DataTables
+      const datatablesSimple = document.getElementById("datatablesSimple");
+      if (datatablesSimple) {
+        new simpleDatatables.DataTable(datatablesSimple);
+      }
 
-        const datatablesSimple = document.getElementById("datatablesSimple");
-        if (datatablesSimple) {
-          new simpleDatatables.DataTable(datatablesSimple);
-        }
-      });
+      // Toggle the side navigation
+      const sidebarToggle = document.querySelector("#sidebarToggle");
+      if (sidebarToggle) {
+        sidebarToggle.addEventListener("click", (event) => {
+          event.preventDefault();
+          document.body.classList.toggle("sb-sidenav-toggled");
+          localStorage.setItem(
+            "sb|sidebar-toggle",
+            document.body.classList.contains("sb-sidenav-toggled")
+          );
+        });

```

Comment:
Page has complete indentation issues. Mixed with space and tabs. Please be sure to sure anyone.